app/build.gradle

```diff
@@ -310,10 +311,11 @@ dependencies {
     implementation 'com.google.firebase:firebase-crashlytics'
     implementation 'com.google.firebase:firebase-crashlytics-ndk'
     implementation('com.google.firebase:firebase-iid:21.1.0')
+    implementation('com.google.firebase:firebase-messaging')
 
     // push notifications
```

Comment:
@josikie Lets add the In-App messaging dep:
implementation("com.google.firebase:firebase-inappmessaging-display")
And remove the duplicate firebase-messaging on line 314` we have the correct one on 319-320

Reply:
ok, noted!